package.json

```diff
@@ -101,7 +104,9 @@
     "analyze": "webpack --config webpack.analyze.js",
     "build": "rimraf dist/ && mkdirp dist && cp README.md dist && rollup -c && webpack",
     "watch": "yarn build --watch",
-    "dist": "yarn test && yarn build && yarn dist:zip",
+    "postversion": "yarn dist && node update-sri.js package README.md contrib/*/README.md docs/*.md website/pages/index.html",
```

Comment:
Does semantic release run this?

Reply:
Yes, after it updates the version but before packing the package.